package-lock.json

```diff
@@ -3741,8 +3741,9 @@
       "dev": true
     },
     "page-metadata-parser": {
-      "version": "github:mozilla/page-metadata-parser#66397fbafb5aab3a7840854d9d3a79c72a9620e6",
-      "from": "github:mozilla/page-metadata-parser#66397fbafb5aab3a7840854d9d3a79c72a9620e6"
+      "version": "1.1.3",
+      "resolved": "https://registry.npmjs.org/page-metadata-parser/-/page-metadata-parser-1.1.3.tgz",
+      "integrity": "sha512-/ILdoE7T62QAqUllDLuZ7mA0x7yYc4S94jjgvz5vxqhYLU2BCUp20SxVir0gq3kezXXAsfLg768UkbvvqYeJ5g=="
```

Comment:
I'm pretty sure this is a valid change but for the sake of this PR can you please remove the changes to this file.